src/tagset.jl

```diff
@@ -3,28 +3,33 @@ const Tag = SmallString
 const maxTagLength = smallLength
 const MTagStorage = MSmallStringStorage # A mutable tag storage
 const IntTag = IntSmallString  # An integer that can be cast to a Tag
+const emptyIntTag = IntTag(0)
```

Comment:
Perhaps you are already planning this change, but probably best to remove this definition entirely in favor of ` emptytag(::Type{IntTag}) = IntTag(0)` that you defined below to avoid unnecessary global variables.

Reply:
Good call, done.